### ISSUE TITLE
Game link redirection for signed-out user

### ISSUE THREAD

**Issue by tomatocat** [googleinterns/step119-2020#37]
**Describe the bug**
When a user is signed out and they try to directly access a game link, they are redirected to the home page with no warning.

**Expected behavior**
Somewhere in the redirection process, the user should be able to see a message telling them that they are being redirected because they must sign in to view game pages.

**Comment by pl98** [comment]
This may need further discussion. This behavior currently exists because we want the user to be smoothly redirected after clicking the sign out button. Is showing a warning every time a user signs out going to be problematic in terms of user experience?

**Comment by tomatocat** [comment]
Yes, there should somehow be a difference between clicking the sign-out button vs. straight up trying to visit the game page while not logged in

**Comment by pl98** [comment]
I have a feeling this might be a little difficult to implement based on how Firebase Auth works with the sign out process. This doesn't seem like a super high priority to me so I'll take a look at this some time next week.

**Comment by tomatocat** [comment]
yup, feel free to add priority tags to the issue. I didn't add the "final presentation" milestone because I think it's not 100% necessary.
